packages/main/src/ToolbarButton.ts

```diff
@@ -26,12 +26,9 @@ import { registerToolbarItem } from "./ToolbarRegistry.js";
  *
  * @constructor
  * @abstract
- * @author SAP SE
- * @alias sap.ui.webc.main.ToolbarButton
- * @extends sap.ui.webc.main.ToolbarItem
- * @tagname ui5-toolbar-button
+ * @extends ToolbarItem
  * @public
- * @implements sap.ui.webc.main.IToolbarItem
```

Comment:
We have to create that interface if it's missing in part of moving to real typescript interfaces

```
class ToolbarButton extends ToolbarItem implements IToolbarItem
```